curriculum/js1/3.js

```diff
@@ -5,8 +5,11 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (num,i=0, newStr="") => {
```

Comment:
i is unnecessary here. How about decrementing num until it hits 0?

Reply:
fixed!